tests/buildtest.cmd

```diff
@@ -41,6 +43,8 @@ goto Usage
 
 if defined _crossgen echo Building tests with CrossGen enabled.&set _buildParameters=%_buildParameters% /p:CrossGen=true
 if defined _priorityvalue echo Building Test Priority %_priorityvalue%&set _buildParameters=%_buildParameters% /p:CLRTestPriorityToBuild=%_priorityvalue%
+if defined _verbosity echo Enabling verbose file logging
+if not defined _verbosity set _verbosity=minimal
```

Comment:
Should the verbosity by default be normal?  Otherwise our log has the same info as the console output